config/cable.yml

```diff
@@ -1,11 +1,11 @@
 development:
-  adapter: redis
+  adapter: async
   url: redis://localhost:6379/1
 
 test:
   adapter: test
 
 production:
-  adapter: redis
+  adapter: async
```

Comment:
The async adapter is intended for development/testing and should not be used in production.